app/assets/javascripts/dashboard/dashboard_task_list_view.js

```diff
@@ -428,6 +494,12 @@ class DashboardTaskListView extends React.PureComponent<Props, State> {
         </h3>
         <div className="clearfix" style={{ margin: "20px 0px" }} />
         {this.renderTaskList()}
+        {
+          ((this.state.showFinishedTasks ? this.state.finishedTasks : this.state.unfinishedTasks).length % pageLength === 0) ? (
```

Comment:
I thought that this is an elegant solution because I do not need to save additional variables, but it fails when you finish a task.

Reply:
Hm, the code itself seems fine. What are the symptoms? Maybe render `(this.state.showFinishedTasks ? this.state.finishedTasks : this.state.unfinishedTasks).length` in an additional line to get some useful debug output?